tests/e2e/test_snowflake_feedback_evaluation.py

```diff
@@ -182,14 +182,11 @@ def test_snowflake_feedback_setup(self) -> None:
         TruBasicApp(
             text_to_text=lambda t: f"returning {t}",
         )
-        # Test stage exists.
+        # Test stage doesn't exist.
```

Comment:
can you expand the comment here a bit to note why we no longer expect a stage as we have conda packages available?

Reply:
After thinking about it, I changed it so that we test via the package import approach since otherwise we may not be testing the right stuff.